packages/rest/src/lib/REST.ts

```diff
@@ -247,7 +247,7 @@ export interface REST {
 		(<S extends string | symbol>(event?: Exclude<S, keyof RestEvents>) => this);
 }
 
-export class REST extends EventEmitter {
+export class REST extends AsyncEventEmitter {
```

Comment:
Should get rid of our EE hacks for exposing events and use the much better types of AEE